### ISSUE TITLE
bug(Icon): Redundant size value

### ISSUE THREAD

**Issue by Tilican** [Semantic-Org/Semantic-UI-React#962]
Hi, there is a little Icon bug

### Steps
Create icons 
```
<Icon name="unhide"/>
<Icon size="medium" name="unhide"/>
<Icon size="large" name="unhide"/>
```

Allow size : mini, tiny, small, medium, large, big, huge, massive

### Expected Result
![image](https://cloud.githubusercontent.com/assets/10628316/20748748/bcdbbd66-b6ef-11e6-90d6-61473fb9a242.png)

### Actual Result
![image](https://cloud.githubusercontent.com/assets/10628316/20748730/a94fa97e-b6ef-11e6-9a6e-f585e0bb946d.png)

### Version
0.61.7

Thx

**Comment by layershifter** [comment]
Thanks for report.

[Looks](https://github.com/Semantic-Org/UI-Icon/blob/master/icon.css#L387) that `Icon` component doesn't support `medium` size at all, so we need update `_meta` on component:

```diff
- size: SUI.SIZES,
+ size: _.without(SUI.SIZES, 'medium'),
```

**Comment by levithomason** [comment]
Fixed in 23c70daf02512a95db05b6f86c1db1a8ef6efa15

**Comment by levithomason** [comment]
For clarity, omitting the `size` prop is the same as using a "medium" sized icon 👍

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.61.8`.
